gum/backend-windows/gummemoryaccessmonitor-windows.c

```diff
@@ -16,6 +18,7 @@
 
 typedef struct _GumLiveRangeDetails GumLiveRangeDetails;
 typedef struct _GumRangeStats GumRangeStats;
+typedef struct _GumPageDetails GumPageDetails;
```

Comment:
Could we move this up so the typedefs follow the same order as the structs they're for?